### ISSUE TITLE
Reproducibility problem

### ISSUE THREAD

**Issue by HumanZhong** [Mountchicken/Union14M#16]
Hi, I'm encountering an accuracy gap during reproducing `mae_rec_base`. The reproduced results in the training log are as follows:

`CUTE80/recog/word_acc: 0.9375  CUTE80/recog/word_acc_ignore_case: 0.9444  CUTE80/recog/word_acc_ignore_case_symbol: 0.9722  IIIT5K/recog/word_acc: 0.5110  IIIT5K/recog/word_acc_ignore_case: 0.8760  IIIT5K/recog/word_acc_ignore_case_symbol: 0.9843  SVT/recog/word_acc: 0.7002  SVT/recog/word_acc_ignore_case: 0.9583  SVT/recog/word_acc_ignore_case_symbol: 0.9706  SVTP/recog/word_acc: 0.6682  SVTP/recog/word_acc_ignore_case: 0.9194  SVTP/recog/word_acc_ignore_case_symbol: 0.9302  IC13/recog/word_acc: 0.4749  IC13/recog/word_acc_ignore_case: 0.9744  IC13/recog/word_acc_ignore_case_symbol: 0.9793  IC15/recog/word_acc: 0.8397  IC15/recog/word_acc_ignore_case: 0.8642  IC15/recog/word_acc_ignore_case_symbol: 0.8844`

There is a 0.5~1.0 gap in almost all datasets compared to your reported results. Do you have any idea what mistake could lead to this?

I also encountered a package conflict following your installation guide. So I made some modifications to avoid conflict and my environment is as follows:

`mmcv==2.0.0rc4`
`mmengine==0.10.2`
`mmdet==3.0.0rc5`
`mmocr==1.0.0`
`cuda==10.1`
`torch==1.8.1+cu101`

Could you also provide your detailed environment so I can check whether the problem is caused by environment difference or not?

**Comment by Mountchicken** [comment]
The results are already close enough. How many epochs did you train?

**Comment by HumanZhong** [comment]
10epochs, I follow the config `maerec_b_union14m.py` and modify the `pretrained` value

**Comment by HumanZhong** [comment]
![image](https://github.com/Mountchicken/Union14M/assets/31882926/ce9d7fe3-6aa1-41eb-88ef-87ab238c826f)
I think the gap is quite significant. Could you please provide your detailed environment? It seems the only difference lies in the environment installation step.

**Comment by Mountchicken** [comment]
I am sorry but I can no longer find the enviroment now, since I have shifed my research interest. The reason of the performance drop can be tricky. I suggest using this training [schedule ](https://github.com/Mountchicken/Union14M/blob/main/mmocr-dev-1.x/configs/textrecog/_base_/schedules/schedule_adamw_cos_6e.py)to see if the performance can get normal.

**Comment by HumanZhong** [comment]
Do you mean [schedule_adamw_cos_6e.py](https://github.com/Mountchicken/Union14M/blob/main/mmocr-dev-1.x/configs/textrecog/_base_/schedules/schedule_adamw_cos_6e.py) may work better than [schedule_adamw_cos_10e.py](https://github.com/Mountchicken/Union14M/blob/main/mmocr-dev-1.x/configs/textrecog/_base_/schedules/schedule_adamw_cos_10e.py)? I will have a try and thank you so much.

**Comment by Mountchicken** [comment]
@HumanZhong 
You can also check this comment:
https://github.com/Mountchicken/Union14M/issues/17#issuecomment-1914523915
